.github/workflows/ci.yml

```diff
@@ -0,0 +1,58 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  pytest:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: False
+      matrix:
+        python_version:
+          - 2.7.13
+          - 3.6.8
+          - 3.7
+          - 3.8
+    steps:
+    - uses: actions/checkout@v2
+    - name: Prepare environment
+      run: .github/workflows/prepare_environment.sh ${{ matrix.python_version }}
+    - name: Run pytest
+      run: |
+        . "${CONDA}/bin/activate" test-env
+        pytest
+
+  pylint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Prepare environment
+      run: .github/workflows/prepare_environment.sh 3
+    - name: Run pylint
+      run: |
+        . "${CONDA}/bin/activate" test-env
+        pylint -E diraccfg/ tests/
+
+  pycodestyle:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Prepare environment
+      run: .github/workflows/prepare_environment.sh 3
+    - name: Run pycodestyle
+      run: |
+        . "${CONDA}/bin/activate" test-env
+        pycodestyle diraccfg/ tests/
+
```

Comment:
pylint and pycodestyle are not run for different versions of python?

Reply:
No but I don't think there is any reason to?

Reply:
pycodestyle not, but pylint?

Reply:
Yeah that's a good point, fixed